src/event.js

```diff
@@ -474,9 +474,10 @@ jQuery.event = {
 		},
 		click: {
 
-			// For checkbox, fire native event so checked state will be right
+			// For checkable types, fire native event so checked state will be right
 			trigger: function() {
-				if ( this.type === "checkbox" && this.click && jQuery.nodeName( this, "input" ) ) {
+				if ( rcheckableType.test( this.type ) &&
+					this.click && jQuery.nodeName( this, "input" ) ) {
```

Comment:
There should be a blank line after the opening brace of the body here, to separate the multi-line condition from its block.